client/src/pages/api/food/index.js

```diff
@@ -0,0 +1,13 @@
+import prisma from "../../../lib/prisma";
+
+export default async function handle(req, res) {
+  const { img, author, title } = req.body;
+  const foodItem = await prisma.food.create({
+    data: {
+      title: title,
+      author: author,
+      img: img,
+    },
+  });
+  res.json(foodItem);
+}
```

Comment:
I think it's time to focus on the app's folder structure for a moment.  This code resides under a root-level folder named `client`, but the code is for the server.  

How do you feel about adding a new folder structure like this:


-client
-server
--api
---food.js

`food.js` would then contain methods that map to the HTTP verbs you're using.  for example `post`, `put`, etc.